index.html

```diff
@@ -1820,13 +1819,13 @@ <h2>Trust Model</h2>
           </li>
           <li>
 The <a>subject</a> and <a>verifier</a> trust the <a>issuer</a> to issue
-true (that is, not false) <a>credentials</a> about the <a>subject</a>, and to
-revoke them quickly when appropriate.
+true (that is, not false) <a>verifiable credential</a> about the <a>subject</a>,
```

Comment:
nope, credentials.

Reply:
Fixed.

Reply:
I do not agree with removing the revocation text. An issuer that says they publish revocation information, but does not publish this revocation information for several years is not trustworthy. Note this is different from an issuer that does not publish revocation information because they issue short lived VCs.

Reply:
I'm not sure which revocation text you are referring to?

Reply:
" and to revoke them quickly when appropriate.". You seem to have deleted this according to github snippet above

Reply:
Ah, OK. It looks like that that text is being cut off in the GitHub snippet. I can assure you though that the text is there and is exactly "... and to revoke them quickly when appropriate."